extends.py

```diff
@@ -0,0 +1,35 @@
+class Car:
+    """
+    This is a car class
+    """
+    nrOfInst = 0
```

Comment:
Please use the variables naming rules: lowercase with words separated by underscores as necessary to improve readability.
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

Reply:
this is an old file, it is out of maintenance :P